plugins/out_s3/s3.c

```diff
@@ -1192,6 +1234,9 @@ static int put_all_chunks(struct flb_s3 *ctx)
     char *buffer = NULL;
     size_t buffer_size;
     int ret;
+    char create_time[20];
+    struct tm now_time;
```

Comment:
do you need this?

Reply:
I forgot to delete them all,  I will delete them